src/shaders/shaders.js

```diff
@@ -96,6 +96,8 @@ precision highp float;
 
 #endif`;
 export const preludeFragPrecisionQualifiers = `
+#extension GL_OES_standard_derivatives : enable
```

Comment:
It might be a better idea to inject that extension string before compiling the shader source [here](https://github.com/mapbox/mapbox-gl-js/blob/main/src/render/program.js#L89), and only when the extension is available for use, e.g. something like:

```js
  const standardDerivativesExt = "#extension GL_OES_standard_derivatives : enable\n"
  const fragmentSource = defines.concat(
      context.extStandardDerivative ? standardDerivativesExt.concat(preludeFragPrecisionQualifiers) : preludeFragPrecisionQualifiers,
      preludeCommonSource,
      prelude.fragmentSource,
      preludeFog.fragmentSource,
      source.fragmentSource).join('\n');
```

Otherwise, this code is used by gl-native and that might impact them.

Reply:
Ah got it, thanks @karimnaaji